.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
I can't merge that, we will make the development experience much worse. Please use the `wrap` mode.
https://github.com/callemall/material-ui/blob/master/.babelrc#L22